piksi_tools/console/utils.py

```diff
@@ -488,11 +488,9 @@
     IMU_CALIBRATION_ERROR: "Cal Error"
 }
 
-SMOOTHPOSE = 0
 DR_RUNNER = 1
 
 ins_type_dict = {
-    SMOOTHPOSE: "SP",
```

Comment:
We should keep this one, if they're using an old firmware, we'd still want to identify that smoothpose is active, see: https://swift-nav.atlassian.net/browse/DEVINFRA-526